runtime/doc/lua.txt

```diff
@@ -595,13 +595,33 @@ vim.highlight.on_yank({opts})                         *vim.highlight.on_yank()*
           - {on_visual} highlight when yanking visual selection (default `true`)
           - {event} event structure (default |v:event|)
 
-vim.highlight.range({bufnr}, {ns}, {higroup}, {start}, {finish}, {rtype}, {inclusive})
+vim.highlight.range({bufnr}, {ns}, {hlgroup}, {start}, {finish}, {opts})
                                                        *vim.highlight.range()*
-        Highlights the range between {start} and {finish} (tuples of {line,col})
-        in buffer {bufnr} with the highlight group {higroup} using the namespace
-        {ns}. Optional arguments are the type of range (characterwise, linewise,
-        or blockwise, see |setreg|; default to characterwise) and whether the
-        range is inclusive (default false).
+
+        Apply highlight group to range of text.
+
+                Parameters: ~
+                    {bufnr}   buffer number
+                    {ns}      namespace for highlights
+                    {hlgroup} hilight group name
```

Comment:
We use `highlight` everywhere else in the docs, so for consistency:

```suggestion
                    {hlgroup} highlight group name
```

Reply:
This doesn't seem to match the doctoring of `M.range` in the Lua file. Do you want to regenerate the docs?

Reply:
These docs are not auto-generated, it looks like. (Probably because the file was too small to merit automation.)